src/utils/client.js

```diff
@@ -1,7 +1,11 @@
 import ApolloClient, { gql } from 'apollo-boost';
 
+export const REACT_APP_API_URL =
```

Comment:
Please, move it to separate merge request, I want  to merge it as quickly as possible